niap-cc/Permissions/Tester/app/src/main/res/mipmap-anydpi-v26/ic_launcher.xml

```diff
@@ -1,20 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<!--
```

Comment:
Nit, here and `ic_launcher_round.xml` below: why was the copyright header removed from this file?

Reply:
added, but it might be removed once we change the application icon.
I added this icon with androdid studio tool.

Reply:
Sounds good, thanks!